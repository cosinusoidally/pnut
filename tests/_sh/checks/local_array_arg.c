// expect_comp_failure
void main() {
  int a[10];
}
