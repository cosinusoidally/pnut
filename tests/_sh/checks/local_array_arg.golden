"a" variable: tests/_sh/checks/local_array_arg.c:5:0  array/struct value type is not supported for shell backend. Use a reference type instead.
